manifests/charts/aeraki/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: aeraki
+description: Manage any layer 7 traffic in Istio service mesh!
+
+# A chart can be either an 'application' or a 'library' chart.
```

Comment:
I suggest we add some introduction here so the user knows how to properly install the required Istio component.

For example:
1. Require Istio 10.0.1 or above
2. Require smart dns enabled
See https://github.com/aeraki-framework/aeraki/blob/master/test/e2e/common/istio-config.yaml for Istio installation configuration

Reply:
smart dns, use helm to manage more appropriate

Reply:
the description  fields Just the introduction function, I think it is more appropriate to explain in the readme.md

Reply:
I think it is necessary to add a copy of how to install aeraki with istio installed,The same question has been raised